HealthcarePortalPrivacyChecklist.md

```diff
@@ -4,3 +4,740 @@
+</p>
+<p>
+The personal information about healthcare providers is collected so they can obtain an account to the portal to access the one-time keys. 
+</p>
+<p>
+Telephone numbers are collected so that a two-factor authentication code can be sent to PT administrators and healthcare providers as a security measure which is required for creating their portal account.  
+</p>
+<p>
+CDS maintains an audit log of activities of both PT administrators and the healthcare providers for activities such as: logging in, inviting new users, account creation, getting a one-time key, etc. The purpose of the audit log is to reduce likelihood of fraudulently generating one-time keys and to trace back anomalous or suspicious behaviour.  
+</p>
+   </td>
+   <td>CDS security/operational staff responsible for managing the portal. 
+   </td>
+   <td>
+<p>
+There are no disclosures of the PT administrator or healthcare provider personal information to any third parties. 
+</p>
+<p>
+In the event that fraudulent or illegal activity is discovered, information from the audit log could be provided to law enforcement. Any such disclosures would be done in accordance with subsection 8(2) of the <em>Privacy Act.</em>
+</p>
+   </td>
+   <td>CDS superadmin accounts can see all users and all their information, with the exception of their passwords. Only authorized CDS staff would be granted superadmin accounts
+   </td>
+   <td>
+<p>
+The personal information about the PT administrators and healthcare provider name, email address, telephone number, province/territory, and login credentials, will be retained for up to 30 days after the pandemic is declared over, at which point the portal will be de-commissioned and the information it stores will be deleted.  
+</p>
+<p>
+PT administrators are responsible for deactivating accounts when they are no longer needed. When an account is deactivated, a record of the account is retained in the audit logs until the portal is de-commissioned in order to maintain a record of deleted accounts (e.g. so that CDS can know if a deleted account is re-created, and can flag if it is suspicious); except for audit logs implicated in a security investigation (if any), which could be retained for an additional 24 months. Access to these audit logs is restricted to CDS security/operational staff.
+</p>
+   </td>
+  </tr>
+</table>
+
+## Risks, mitigation and safeguards
+
+<table>
+  <tr>
+   <td>
+   </td>
+   <td><strong>Yes</strong>
+   </td>
+   <td><strong>No</strong>
+   </td>
+   <td><strong>N/A</strong>
+   </td>
+  </tr>
+  <tr>
+   <td>Are staff is appropriately trained on privacy requirements?
+   </td>
+   <td colspan="2" >Partially 
+   </td>
+   <td>☐
+   </td>
+  </tr>
+  <tr>
+   <td>Has the appropriate <a href="http://mysource.hc-sc.gc.ca/eng/ss/programs-services/information-management-information-technology/it-security/care-and-custody">security classification</a> been applied to the personal information?
```

Comment:
Looks like this link is available internally only, is there a different one we should point to, or note that it's internal only?